test/tests/ver-1dc/mms.i

```diff
@@ -0,0 +1,233 @@
+cl = 1
```

Comment:
Please add a comment header describing what this input file is intended to do, similar to many of our other cases.